.github/workflows/lint_legacy_python.yml

```
@@ -0,0 +1,19 @@
+name: lint_legacy_python
+on: [pull_request, push]
+jobs:
+   lint_legacy_python:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: actions/checkout@v2
+       - uses: actions/setup-python@v3
+         with:
+           python-version: 2.7
+       - run: sudo apt upgrade -y && sudo apt-get install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils -y 
```

```suggestion
       - run: |
           sudo apt-get -q update
           sudo apt-get -qq upgrade
           sudo apt-get -qq install curl ffmpeg libcurl4-openssl-dev libssl-dev motion ssh v4l-utils
```
- `apt-get` is better suited for non-interactive/script executions.
- `-qq` implies `-y` and produces less processing output which isn't nice to read in GitHub actions logs.